src/raster/gcp.rs

```diff
@@ -20,11 +123,83 @@ impl Dataset {
 
         unsafe { SpatialRef::from_c_obj(c_ptr) }.ok()
     }
+
```

Comment:
```suggestion
    /// Get the projection definition string for the GCPs in this dataset.
    ///
    /// # Notes
    /// * This is separate and distinct from [`Dataset::projection`], and only applies to
    /// embedded GCPs.
    ///
    ///  See: [`GDALGetGCPProjection`](https://gdal.org/api/raster_c_api.html#gdal_8h_1a85ffa184d3ecb7c0a59a66096b22b2ec)
    pub fn gcp_projection(&self) -> Option<String> {
        let cc_ptr = unsafe { gdal_sys::GDALGetGCPProjection(self.c_dataset()) };
        if cc_ptr.is_null() {
            return None;
        }
        Some(_string(cc_ptr))
    }
```

If you merge this I'll close https://github.com/georust/gdal/pull/402

Reply:
Done, though we should have probably merged #402 directly instead :smile:.

Reply:
🙏